app/pb_kits/playbook/pb_body/_body.html.erb

```diff
@@ -1,5 +1,6 @@
-<%= content_tag(:"#{object.tag}", object.text,
-    id: object.id,
-    data: object.data,
-    class: object.classname("body#{object.color}#{object.dark}"),
-    aria: object.aria) %>
+<%= content_tag(object.tag,
+  id: object.id,
+  data: object.data,
+  class: object.classname("body#{object.color}#{object.dark}")) do %>
+ <div><%= object.yield(context: self) %></div>
```

Comment:
I don't think we need this wrapping <div></div>. They can pass in div's, p's, ect

Reply:
Are you asking to remove line 5?